forms-shared/src/summary-email/renderSummaryEmail.tsx

```diff
@@ -8,7 +8,7 @@ import { render } from '@react-email/components'
 import { SummaryEmail } from './SummaryEmail'
 import { FormDefinition } from '../definitions/formDefinitionTypes'
 
-export type RenderSummaryEmailFileIdUrlMap = Record<string, string>
+export type RenderSummaryEmailFileIdUrlMap = Record<string, { url: string; fileName: string }>
```

Comment:
I would maybe change the naming to something like `RenderSummaryEmailFileIdInfoMap`, since now it contains file name as well.